mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss

```diff
@@ -1,18 +1,114 @@
 @import "src/styles/variables";
 
+$maxWidth: 1200px;
+
 .pageWrapper {
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 0 $gapMedium;
   margin-top: $marginMedium;
+  gap: $gapBig;
+}
+
+.title {
+  color: var(--secondaryTextButtonColor);
+  text-align: center;
+}
+
+.subtitle {
+  color: var(--primaryTextColor);
+  text-align: center;
+  margin-bottom: $marginSmall;
+}
+
+.aboutBlock {
+  max-width: $maxWidth;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: $gapSmall;
+  margin-top: $marginBig;
+
+  color: var(--secondaryTextColor);
+  background-color: var(primarysBackgroundColor);
+}
+
+.aboutDescription {
+  display: flex;
+  align-items: center;
+  margin-top: $marginBig;
+  background-color: var(--primaryBackgroundColor);
+}
+
+.aboutTextBlock {
+  flex: 1 1 70%;
+  padding: $paddingMedium;
+  display: flex;
+  flex-direction: column;
+  gap: $gapBig;
+  line-height: 1.5;
```

Comment:
different line height looks strange why do we need this? lets try to avoid

Reply:
its better for desigh with line-height
![image](https://github.com/tritonJS826/masters-way/assets/112805668/e8605e14-f7f9-45b4-b48e-0e6e444ecb58)
without
![image](https://github.com/tritonJS826/masters-way/assets/112805668/8d31a7d5-7012-4bc1-b0cb-da0f84c234ed)

Reply:
I'm talking about consistency not about specific case of view. If you created two extra line heights for just this page and this line height are defined locally. So in feature it will be hard to have consistent design on the site. Take some time thinking about styles consistency and we can discuss it if it is not clear.